data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escap_attempts, neutered, weight_kg)
```

Comment:
- [ ] Kindly fix this typo error in the insert statement for the `escap_attempts` as it's called `escape_attempt` in your schema.sql file.